src/goto-cc/gcc_mode.cpp

```diff
@@ -61,6 +61,9 @@ static std::string compiler_name(
      base_name.find("goto-bcc")!=std::string::npos)
     return "bcc";
 
+  if(base_name=="goto-clang")
+    return "clang";
```

Comment:
Can we thus get rid of the `#ifdef __FreeBSD__` below?

Reply:
No, unfortunately not -- I'd like that goto-cc (with that basename) will call clang on FreeBSD. I don't see any way of achieving that without the #ifdef.